util.cpp

```diff
@@ -79,6 +79,29 @@ namespace Sass {
     return out;
   }
 
+  // read css string (handle multiline DELIM)
+  string read_css_string(const string& str)
+  {
+    string out("");
+    bool esc = false;
+    for (auto i : str) {
+      if (i == '\\') {
+        esc = ! esc;
+      } else if (esc && i == '\r') {
```

Comment:
Do we need to worry about platform difference here? Are `\r\n` line ending going to cause this to back track too many times?